### DISCUSSION TITLE
Emit after TextInputColumn update in RelationManager

### DISCUSSION THREAD

**Discussion by AugustBurnsRed** [filamentphp/filament#8464] [Help] [answered]
Hi, i have a widget with a value that need to be updated after i change the content of a `TextInputColumn ` in my table. The table is in a RelationManager.

I was trying to use `updateStateUsing` for catching when the value is updated and then trigger an emit(). However it's not working `TransactionsRelationManager::emit does not exist.`

Thanks!

```
TextInputColumn::make('amount')
                    ->sortable()
                    ->type('number')
                    ->updateStateUsing(function ($livewire) {
                        $livewire->emit('updateAccountOverview');
                    }),
```

**Comment by leandrocfe** [comment, marked as answer]
livewire 3? [dispatch](https://livewire.laravel.com/docs/events#dispatching-events)
